homeassistant/components/onewire/sensor.py

```diff
@@ -92,15 +103,17 @@ def hb_info_from_type(dev_type="std"):
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the one wire Sensors."""
-    base_dir = config[CONF_MOUNT_DIR]
-    owport = config[CONF_PORT]
+    base_dir = config.get(CONF_MOUNT_DIR)
```

Comment:
Please revert these changes. There are defaults in place from the platform schema.